pom.xml

```diff
@@ -46,6 +46,7 @@
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
     <quarkus.platform.version>3.15.2</quarkus.platform.version>
     <quarkus-quinoa.version>2.5.1</quarkus-quinoa.version>
+    <org.hibernate.orm.hibernate.jfr.version>6.6.4.Final</org.hibernate.orm.hibernate.jfr.version><!-- TODO is there some way to grab the Hibernate version used by Quarkus to match this version? -->
```

Comment:
I opened a discussion about this here: https://github.com/quarkusio/quarkus/discussions/45552

Reply:
Quarkus 3.15.2 with the Panache/Hibernate extension pulls in Hibernate 6.6.1, which I originally used in this PR to keep the Hibernate JFR version in sync. But, it runs into https://github.com/hibernate/hibernate-orm/pull/9180 , so I upgraded to a later bugfix version to work around this.

Reply:
Quarkus 3.15.3 uses Hibernate 6.6.3, which also contains the fix.